server/src/main/java/io/crate/types/DataType.java

```diff
@@ -71,9 +71,9 @@ public enum Precedence {
         BOOLEAN,
         SHORT,
         TIMETZ,
+        REGCLASS,
         INTEGER,
```

Comment:
```
ERROR: Unknown function: pg_catalog.pg_get_expr(d.adbin, d.adrelid), no overload found for matching argument types: (text, regclass). Possible candidates: pg_catalog.pg_get_expr(text, integer):text, pg_catalog.pg_get_expr(text, integer, boolean):text
```
There are hundreds of this failures, need to allow implicit cast from REGCLASS to INTEGER.